mrp_multi_level/wizards/mrp_multi_level.py

```diff
@@ -538,6 +538,16 @@ def _mrp_initialisation(self, mrp_areas):
                 self._init_mrp_move(product_mrp_area)
         logger.info("End MRP initialisation")
 
+    def _get_qty_to_order(self, product_mrp_area, date, move_qty, onhand):
+        """compute the qty to order at a given date, for a product MRP area, given an
+        mrp.move quantity and an onhand quantity.
+
+        The method takes into account the safety stock priorization and safety
```

Comment:
This text is not relevant any more, right?

Reply:
you're right. Fixed in d45721609ced96f652d83f02ec6c53d37e36b4c9

Happy new year to the ForgeFlow team :grin: